src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -38,7 +38,7 @@
 	private static final long serialVersionUID = -772706090092654841L;
 	
 	JPanel infoPanel_;
-    JPanel datePanel_;
+    JPanel datePanel2_;
     Box buttonBox_;
 
     JButton nextButton_;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "nextButton_" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3AS00116)